### PR TITLE
Remove box shadow for ha-card

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

After discussing with @matthiasdebaat. We removed default box-shadow and increase default border radius for all `ha-card`. Another PR will follow to remove all the `outlined` properties as it's the default.

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-card.ts [hunk 1]
```diff
@@ -14,26 +14,26 @@ export class HaCard extends LitElement {
           --ha-card-background,
           var(--card-background-color, white)
         );
-        border-radius: var(--ha-card-border-radius, 4px);
-        box-shadow: var(
```
Comment (bramkragten) [inline src/components/ha-card.ts:18]: removing this makes that it is no longer themeable and thus a breaking change, is that intended?

Comment (piitaya) [reply src/components/ha-card.ts:18]: It's in the raised option. If we let this by default, all themes that override `ha-card-box-shadow` will have border + box shadow. 
I don't known what is the better option

File: src/components/date-range-picker.ts [hunk 2]
```diff
@@ -111,14 +111,14 @@ class DateRangePickerElement extends WrappedElement {
           .daterangepicker {
             left: 0px !important;
             top: auto;
+            box-shadow: var(--ha-card-box-shadow, "none");
```
Comment (bramkragten) [inline src/components/date-range-picker.ts:114]:
```suggestion
            box-shadow: var(--ha-card-box-shadow, none);
```

File: src/components/ha-card.ts [hunk 3]
```diff
@@ -14,26 +14,28 @@ export class HaCard extends LitElement {
...
-          --ha-card-box-shadow,
-          0px 2px 1px -1px rgba(0, 0, 0, 0.2),
-          0px 1px 1px 0px rgba(0, 0, 0, 0.14),
-          0px 1px 3px 0px rgba(0, 0, 0, 0.12)
+        box-shadow: var(--ha-card-box-shadow, "none");
```
Comment (bramkragten) [inline src/components/ha-card.ts:17]:
```suggestion
        box-shadow: var(--ha-card-box-shadow, none);
```

**Review by bramkragten — DISMISSED** [review]

**Review by bramkragten — APPROVED** [review]

**Comment by krazos** [comment]
Maybe it's just me, but in edit mode, the edit bar looks a little odd with the squared-off top border and rounded bottom border after these changes to the border radius: 

image

I'm by no means a design or UX expert, but it might be worth considering additional tweaks to the way the edit bar is displayed.

**Pull request merged**